frontend/javascripts/messages.js

```diff
@@ -71,11 +71,10 @@ In order to restore the current window, a reload is necessary.`,
   "download.close_window": "You may close this window after the download has started.",
   "add_script.confirm_change": "This will replace the code you have written. Continue?",
   "data.enabled_render_missing_data_black":
-    "You just enabled the option to render missing data black. Please refresh this page so that the changes can take effect.",
-  "data.disabled_render_missing_data_black": `You just disabled the option to render missing
+    "You just enabled the option to render missing data black. All layers will now be reloaded.",
+  "data.disabled_render_missing_data_black": `  You just disabled the option to render missing
```

Comment:
I added these two spaces to let the whole text have an equal indent. Previously, the first line in the info toast was two spaces more left than the other two lines which did not look nice.

Reply:
Just chiming in for this comment: I think it makes more sense to remove the other leading spaces in the other two lines. There were probably added by accident :)